secure_message/repository/retriever.py

```diff
@@ -33,10 +33,18 @@ def unread_message_count(user):
     @staticmethod
     def message_count_by_survey(user, survey, label=None):
         """Count users messages for a specific survey"""
+
+        survey_conditions = []
+
+        if survey:
+            survey_conditions.append(SecureMessage.survey.in_(survey))
+        else:
+            survey_conditions.append(True)
+
         if user.is_internal:
-            status_conditions, survey_conditions = Retriever._get_conditions_internal_user(survey, user)
+            status_conditions = []
```

Comment:
Could you not set the status conditions like you have survey_conditions prior and reduce the need for the else? i.e. `if !user.is_internal: status_conditions = [Status.actor == str(user.user_uuid)]`